fury/primitive.py

```diff
@@ -851,3 +852,195 @@ def prim_cylinder(radius=0.5, height=1, sectors=36, capped=True):
         triangles = (np.array(triangles).reshape(2 * sectors, 3))
 
     return vertices, triangles
+
+
+def build_parametric(u_lower_bound, u_upper_bound, v_lower_bound,
+                     v_upper_bound, npoints, surface_equation):
```

Comment:
can you add a docstring for this function

Reply:
Done